api/src/main/java/org/openmrs/module/aijar/metadata/core/PublicHolidays.java

```diff
@@ -0,0 +1,56 @@
+package org.openmrs.module.aijar.metadata.core;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.openmrs.BaseOpenmrsData;
+import org.springframework.stereotype.Component;
+
+@Component
+@Entity(name = "ugandaemr.public_holiday")
```

Comment:
@reagan-meant what is the ugandaemr. prefix?

Reply:
Done